<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AvatarSettings</class>
 <widget class="QWidget" name="AvatarSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="layout">
   <property name="sizeConstraint">
    <enum>QLayout::SetNoConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item row="2" column="1">
    <widget class="QComboBox" name="cbxType">
     <item>
      <property name="text">
       <string>Letter</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Icon</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Pixmap</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="sbxSize">
     <property name="minimum">
      <number>5</number>
     </property>
     <property name="maximum">
      <number>300</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblType">
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblSize">
     <property name="text">
      <string>Size</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QCheckBox" name="chkDisabled">
     <property name="text">
      <string>Disabled</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="MaterialAvatar" name="avatar">
     <property name="letter">
      <char>
       <unicode>88</unicode>
      </char>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MaterialAvatar</class>
   <extends>QWidget</extends>
   <header location="global">QtMaterialWidgets/avatar.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
